jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildConfiguration.java

```diff
@@ -425,6 +444,15 @@ public boolean isOffline() {
     return offline;
   }
 
+  /**
+   * Gets whether or not to cache the base images.
```

Comment:
The doc is misleading. I think "... whether or not to force caching ..." will do.

Reply:
Changed it.